src/Routing/Router.php

```diff
@@ -825,7 +825,7 @@ public static function reverseToArray($params)
      * are used for CakePHP internals and should not normally be part of an output URL.
      *
      * @param \Cake\Http\ServerRequest|array $params The params array or
-     *     Cake\Network\Request object that needs to be reversed.
+     *     Cake\Http\ServerRequest object that needs to be reversed.
```

Comment:
We should backport this already to 3.6 IMO.

Reply:
done.